src/Symfony/Component/RateLimiter/Policy/TokenBucket.php

```diff
@@ -98,6 +98,18 @@ public function __serialize(): array
 
     public function __unserialize(array $data): void
     {
+        // BC layer for old objects serialized via __sleep
+        if (5 === \count($data)) {
+            $data = array_values($data);
+            $this->id = $data[0];
+            $this->tokens = $data[1];
+            $this->timer = $data[2];
+            $this->burstSize = $data[3];
+            $this->stringRate = $data[4];
```

Comment:
actualy let's remove this one, remove wakeup and move its logic here

Reply:
argh no! that'd break compat with php < 7.4

Reply:
`$this->rate = Rate::fromString($data[4]);` then

Reply:
Oh yeah missed that the stringRate prop does not exist :)